packages/vue-components/src/Navbar.vue

```diff
@@ -318,8 +332,10 @@ export default {
 
         .navbar-default > ul > * {
             padding: 0.3125rem 0.625rem;
-            flex: 1;
-            background: rgba(0, 0, 0, 0.3);
```

Comment:
👀

> ok, I see the point now, I think this works best in that case.

I actually liked the old approach with setting backgroud colour on the individual `ul>*`'s  https://github.com/MarkBind/markbind/pull/1690#discussion_r771130744. (just need a separate `.navbarlight .navbar-default > ul > * { background: ... }` of sorts)

I think dynamic style detection (`--mobile-link-highlight-color`) is something we should avoid when possible as it may introduce more noticeable / distracting visual changes after the page has been presented already. (try seeing the netlify on a slow enough mobile device, you'll get to see the background colour change)

Reply:
Alright I've changed back to the previous implementation. Initially when I flushed all nav items to the left https://github.com/MarkBind/markbind/pull/1690#discussion_r771835529, it left some spaces at the end on some screen sizes so I decided to apply the colour to the entire background instead.

![image](https://user-images.githubusercontent.com/47470981/147409158-77e56ba8-9250-4117-94cd-2047579df54a.png)

I tried playing around with the flex property and decided to go with `flex-grow: 1` so that the nav items fill the bar completely and also ensure that the spacing between each item is "equal". I think this allows for setting the background of each individual nav link as well as addresses the spacing issue.

![image](https://user-images.githubusercontent.com/47470981/147409196-3cefaa2b-1946-4545-982e-5b808aa873bc.png)